tests/check/check_signed_video_auth.c

```diff
@@ -733,32 +654,20 @@ START_TEST(remove_the_i_nalu)
   signed_video_accumulated_validation_t final_validation = {
       SV_AUTH_RESULT_NOT_OK, false, 24, 23, 1, SV_PUBKEY_VALIDATION_NOT_FEASIBLE, true, 0, 0};
   // A missing I NAL unit will affect two GOPs, as it is part of both the gop_hash and the
-  // linked_hash. At the GOP level, the missing NAL unit will invalidate both GOPs. However, at the
-  // Frame level, the second GOP, which is invalid at the GOP level, is considered valid because the
-  // previous GOP link has no effect on Frame level validation.
-  // IPPSIPPSPPSIPPSI
+  // linked_hash. At the GOP level, the missing NAL unit will invalidate both GOPs.
+  // IPPSIPPSPPSIPPSIPPSIPPSI
   //
   //  IPPS                      ->   (valid) -> ....
   //      IPPSP                 ->   (valid) -> ....
```

Comment:
There are some errors here in validation stream. Not aligned with the total stream above and some weird ends for example here, including a "P"